From 6a9f5a9abe88d3efbe97a23b8cd044756dcf2d5a Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Thu, 16 Jul 2015 17:45:14 +0800
Subject: [PATCH] eDP:rk3288:ignore grf probe error when probe.

rockhip,grf node is not exist on rk3288,  we
need to ignore grf probe error.

Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/video/rockchip/transmitter/rk32_dp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/transmitter/rk32_dp.c b/drivers/video/rockchip/transmitter/rk32_dp.c
index 378c4e7d12dc..c04e6c94e220 100755
--- a/drivers/video/rockchip/transmitter/rk32_dp.c
+++ b/drivers/video/rockchip/transmitter/rk32_dp.c
@@ -1428,7 +1428,7 @@ static int rk32_edp_probe(struct platform_device *pdev)
 	}
 
 	edp->grf = syscon_regmap_lookup_by_phandle(np, "rockchip,grf");
-	if (IS_ERR(edp->grf)) {
+	if (IS_ERR(edp->grf) && !cpu_is_rk3288()) {
 		dev_err(&pdev->dev, "can't find rockchip,grf property\n");
 		return PTR_ERR(edp->grf);
 	}
-- 
2.35.3

